app/src/main/java/ai/elimu/appstore/BaseApplication.java

```diff
@@ -92,13 +102,55 @@ public Retrofit getRetrofit() {
         HttpLoggingInterceptor logging = new HttpLoggingInterceptor();
         logging.setLevel(BuildConfig.DEBUG ? HttpLoggingInterceptor.Level.BODY :
                 HttpLoggingInterceptor.Level.NONE);
-        final OkHttpClient okHttpClient = new OkHttpClient.Builder()
+
+        final ProgressResponseBody.ProgressListener progressListener = new ProgressResponseBody.ProgressListener() {
+            @Override
+            public void update(long bytesRead, long contentLength, boolean done) {
+//                Timber.i("bytesRead: " + bytesRead);
+//                Timber.i("contentLength: " + contentLength);
+//                Timber.i("done: " + done);
+//                Timber.i("%d%% done\n", (100 * bytesRead) / contentLength);
```

Comment:
Are these the lines that caused the OutOfMemoryException?

Reply:
No. But they are unnecessary IMHO, so just disabled them.